scripts/configurator.sh

```diff
@@ -1,11 +1,11 @@
-#!/bin/sh
+#!/bin/bash
 if [[ "$#" -ne 3 ]]
 then 
   echo "Usage: sh configurator.sh CONFIG_PATH VARIABLE_NAME VARIABLE_VALUE"
   exit 1
 fi
 
 
-sed -i.bak "s/$2\ =.*/$2\ =\ $3/g" $1
+sed -i.bak "s!#*.*$2\ =.*!$2\ =\ $3!g" $1
```

Comment:
Changed the delimiter char so that we can use this to insert paths (which contain / chars) into the config.